servers/curated-corpus-api/src/database/queries/Section.ts

```diff
@@ -16,21 +17,39 @@ export async function getSectionsWithSectionItems(
   db: PrismaClient,
   isPublicContext: boolean,
   scheduledSurfaceGuid: string,
-  createSource?: 'ML' | 'MANUAL'
+  createSource?: 'ML' | 'MANUAL',
 ): Promise<Section[]> {
+  const currentDate = DateTime.utc().startOf('day').toJSDate();
+
+  const filterLiveSections = {
+    OR: [
+      // ML sections (no startDate or endDate)
+      { startDate: null, endDate: null },
+      {
+        // Custom Sections that are currently live
+        // a. startDate <= currentDate
+        startDate: { lte: currentDate },
+        OR: [
+          // b. no endDate
+          { endDate: null },
+          // c. currentDate < endDate
+          { endDate: { gt: currentDate } },
+        ],
+      },
+    ],
+  };
+
   const sections = await db.section.findMany({
     where: {
-      scheduledSurfaceGuid: scheduledSurfaceGuid,
+      scheduledSurfaceGuid,
       active: true,
-      // if public query, filter by disabled
-      ...(isPublicContext ? { disabled: false } : {}),
-      // if createSource is provided, filter by it
+      ...(isPublicContext ? { disabled: false, ...filterLiveSections } : {}),
```

Comment:
i _think_ the above could be simplified to:

`...(isPublicContext && { disabled: false, ...filterLiveSections }),`

that way you don't have to pass in an empty object if `isPublicContext` is `false`.

Reply:
oh, and even one step further if you wanted to.

put `disabled: false` inside `filterLiveSections` and then:

`...(isPublicContext && filterLiveSections),`

would save a few cycles of not having to spread the `filterLiveSections` object each time.